scripting/get5.sp

```diff
@@ -173,6 +174,9 @@ bool g_PendingSideSwap = false;
 Handle g_KnifeChangedCvars = INVALID_HANDLE;
 Handle g_MatchConfigChangedCvars = INVALID_HANDLE;
 
+/** Informational data **/
+int g_ClientsConnected = 0; // Counting on OnClientAuthorized and Event_PlayerDisconnect are fired only once for each client
```

Comment:
I don't see the value in this variable - I'd rather a function just count the number of clients connected instead of maintaining this count.

Reply:
I don't know where to place this function and how to get the number of connected clients. Could anyone help me on this?

Reply:
This could be usefull:
https://github.com/splewis/get5/blob/69f6135548dfbbf199f6356691cee69199729dbf/scripting/get5/teamlogic.sp#L225